pom.xml

```diff
@@ -15,13 +16,31 @@
 	<properties>
 		<java.version>1.8</java.version>
 		<org.mapstruct.version>1.3.0.Final</org.mapstruct.version>
-		<jacoco-maven-plugin.version>0.7.7.201606060606</jacoco-maven-plugin.version>
+		<jacoco-maven-plugin.version>0.8.4</jacoco-maven-plugin.version>
```

Comment:
No need for this it will be automatically inherited.

Reply:
removed